toolkit/src/components/cut/metadata/service-health-badge/index.hbs

```diff
@@ -0,0 +1,37 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+}}
+
+<Hds::TooltipButton 
+  @text={{if 
+    this.statusIsCritical '1 or more instances is critical'
+    (if this.statusIsWarning '1 or more instances has a warning'
+    'All instances are healthy')
```

Comment:
Do we want to set up translations for these?

Reply:
We've got a separate ticket for adding translations to the toolkit, it's not currently set up at all. Good callout 👍